description: remove some prune causing build to fail, avoid matching Android namespace declaration/com.google.common package in .java
author: Wengling Chen <feiyu2817@gmail.com>


--- a/ungoogled-chromium/pruning.list
+++ b/ungoogled-chromium/pruning.list
@@ -14,7 +14,6 @@ base/test/data/pe_image/pe_image_test_32.dll
 base/test/data/pe_image/pe_image_test_64.dll
 base/test/data/pe_image/pe_image_test_arm64.dll
 build/android/CheckInstallApk-debug.apk
-build/android/chromium-debug.keystore
 build/android/tests/symbolize/liba.so
 build/android/tests/symbolize/libb.so
 build/linux/sysroot_scripts/debian_archive_unstable.gpg
@@ -14727,12 +14725,9 @@ third_party/hunspell/fuzz/bdict_corpus/en-US-7-1.bdic
 third_party/hunspell/fuzz/bdict_corpus/en-US-8-0.bdic
 third_party/hunspell/fuzz/bdict_corpus/f95048cdc0fc231dd712bad106578a9eeecd869b
 third_party/hunspell/fuzz/bdict_corpus/fda69541054125551c41ab3e591e0482405ca2dc
-third_party/icu/android/icudtl.dat
-third_party/icu/android_small/icudtl.dat
 third_party/icu/android_small/icudtl_extra.dat
 third_party/icu/cast/icudtl.dat
 third_party/icu/chromeos/icudtl.dat
-third_party/icu/common/icudtb.dat
 third_party/icu/flutter/icudtl.dat
 third_party/icu/ios/icudtl.dat
 third_party/icu/source/data/in/coll/ucadata-implicithan.icu
